public/video-ui/src/services/UploadsApi.js

```diff
@@ -5,6 +5,18 @@ import { errorDetails } from '../util/errorDetails';
 import 'aws-sdk/dist/aws-sdk';
 const AWS = window.AWS;
 
+// The timeout for individual upload requests. Defaults to 120s. This
+// default causes problems on slow connections – for example, w/ a 0.5mbps
+// upload speed (3.75MB/minute), uploads for 8mb chunks will never complete.
+AWS.config.httpOptions.timeout = 240_000; // in ms
```

Comment:
why the underscore?

Reply:
Arguably easier to read than `240000`. https://v8.dev/features/numeric-separators